lib/widgets/picos_chart_column.dart

```diff
@@ -0,0 +1,154 @@
+            )
+          : (dataList as List<Daily>?)?.firstWhereOrNull(
+              (Daily daily) => PicosChartHelper.isSameDay(daily.date, date),
+            );
+
+      double? value;
+      if (valuesChartOptions == ValuesChartOptions.walkingDistance) {
+        value = (matchingData as Weekly?)?.walkingDistance?.toDouble();
+      } else if (valuesChartOptions == ValuesChartOptions.bloodSugar) {
+        value = (matchingData as Daily?)?.bloodSugar?.toDouble();
+      } else if (valuesChartOptions == ValuesChartOptions.sleepDuration) {
+        value = (matchingData as Daily?)?.sleepDuration?.toDouble();
+      }
+
+      return PicosChartSampleData(
+        x: dayShortText,
+        y: value,
+      );
+    }).toList();
+  }
+
+  String _formatDateRange(DateTime startDate, DateTime endDate) {
+    final DateFormat formatterStartDate = DateFormat('dd.MM.');
+    final DateFormat formatterEndDate = DateFormat('dd.MM.yyyy');
+    return '$title  ${formatterStartDate.format(startDate)}- '
+        '${formatterEndDate.format(endDate)}';
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
+    TooltipBehavior tooltipBehavior = TooltipBehavior(
+      enable: true,
+      textStyle: const TextStyle(
+        fontSize: kIsWeb ? 18 : 14,
+      ),
+    );
+    final DateTime today = DateTime.now();
+    final DateTime startDate = today.subtract(
+      Duration(days: isWeekly == true ? 42 : 7),
+    );
+
+    final String titleText = _formatDateRange(startDate, today);
+
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(color: theme.blue!, width: PicosChartHelper.width),
+        borderRadius: BorderRadius.circular(10),
+      ),
+      child: SfCartesianChart(
+        plotAreaBorderWidth: 0,
+        title: ChartTitle(
+          text: titleText,
+          textStyle: TextStyle(
+            color: theme.blue,
+            fontWeight: FontWeight.bold,
+            fontSize: 14,
+          ),
+          alignment: ChartAlignment.near,
+        ),
+        primaryXAxis: CategoryAxis(
+          majorGridLines: const MajorGridLines(width: 0),
+          axisLine: AxisLine(
+            color: theme.blue,
+          ),
+          labelStyle: const TextStyle(
+            color: PicosChartHelper.colorBlack,
+          ),
+        ),
+        primaryYAxis: NumericAxis(isVisible: false),
+        tooltipBehavior: tooltipBehavior,
+        series: <ColumnSeries<PicosChartSampleData, String>>[
+          ColumnSeries<PicosChartSampleData, String>(
+            dataSource: _prepareChartData(),
+            xValueMapper: (PicosChartSampleData point, _) => point.x,
+            yValueMapper: (PicosChartSampleData point, _) => point.y,
+            name: title,
+            dataLabelSettings: const DataLabelSettings(
+              isVisible: true,
+              labelAlignment: ChartDataLabelAlignment.outer,
+            ),
+          )
```

Comment:
Can you put a trailing comma

Reply:
My ide doesn't show me any warnings about this, but if it helps, I'll do it.

Reply:
resolved in [refactoring (trailing comma)](https://github.com/healthcare-it-solutions/picos/pull/184/commits/51cfaea476814bf6d057b648b4136a886434d7de)